lib/utils/cn.ts

```diff
@@ -0,0 +1,6 @@
+import clsx, { ClassValue } from "clsx";
```

Comment:
```suggestion
import clsx, { type ClassValue } from "clsx";
```

Let's always do this when we're importing a type from libraries, it's kinda neat and communicates that it a `type` import.

😉

Reply:
Another thing we could do is add verbatim resolution option to our tsconfig so that this happens automatically via auto import.

I have it on all my projects to avoid doing this manually

Reply:
Commit this as well if you can

Reply:
> Another thing we could do is add verbatim resolution option to our tsconfig so that this happens automatically via auto import.

Like that adds the `type` keyword to the types imports, you mean? 😮

Reply:
> Like that adds the `type` keyword to the types imports, you mean? 😮

Cool stuff innit?
All you gotta is turn this not-so-known flag on in the tsconfig:
https://www.typescriptlang.org/tsconfig#verbatimModuleSyntax

Reply:
Haha 😃, completely new to me and sure looks sweet 😋 

Let's hold put this on our thoughts, we will look to implementing later on in the project, It should make sense for DX. In the meantime I will also try use it in some of my project to learn more about it.

Reply:
👍